### DISCUSSION TITLE
关于 X6 库依赖 jquery

### DISCUSSION THREAD

**Discussion by zengyuancheng** [antvis/X6#2153] [Q&A] [answered]
在 X6 的依赖里发现了对 jquery 和 jquery-mousewheel 的依赖，后续升级版本考虑用其他库或者是纯 js 替换 jquery 吗？

**Comment by NewByVector** [comment, marked as answer]
2.0 开发过程中，会替换掉的，可以看[这里](https://www.yuque.com/antv/x6/sk3zdn)
